CMakeLists.txt

```diff
@@ -104,30 +104,18 @@ if(NOT RANGE_V3_NEW_VERSION_HPP STREQUAL RANGE_V3_OLD_VERSION_HPP)
     DESTINATION ${CMAKE_CURRENT_SOURCE_DIR}/include/range/v3
   )
   # Update the conanfiles, too
-  configure_file(conanfile.py.in ${CMAKE_CURRENT_SOURCE_DIR}/conanfile.py @ONLY)
-  configure_file(test_package/conanfile.py.in ${CMAKE_CURRENT_SOURCE_DIR}/test_package/conanfile.py @ONLY)
-  execute_process(
-    COMMAND ${GIT_EXECUTABLE} -C "${CMAKE_CURRENT_SOURCE_DIR}" add -u
-  )
-  execute_process(
-    COMMAND ${GIT_EXECUTABLE} -C "${CMAKE_CURRENT_SOURCE_DIR}" commit --amend --no-edit
-  )
-  execute_process(
-    COMMAND ${GIT_EXECUTABLE} -C "${CMAKE_CURRENT_SOURCE_DIR}" tag -f -a "${RANGE_V3_VERSION}" -m "${RANGE_V3_VERSION}"
-  )
```

Comment:
These three `execute_process` commands are important and have nothing to do with conan. Please reinstate them.

Reply:
Sorry, my mistake, I quickly assumed they were related to the commit of the generated conanfiles, without carefully reading them. Fixed.